characters.php

```diff
@@ -59,8 +59,22 @@ function getTitle()
                 <tbody>
                 <?php
                 $counter = 1;
+
+                /* house kontrolu */
+                $isHouse = false;
+                $house = '';
+                if (isset($_GET['house']) && in_array($_GET['house'],
+                        array('hufflepuff', 'gryffindor', 'ravenclaw', 'slytherin'))) {
```

Comment:
Selamlar, ileride `hufflepuff` adı `huffle` olarak değiştirilse burayı bulmakta biraz zorlanmaz mıyız :) Config içine veya en yukarı const olarak tanımalarsak çok tatlı olacaktır.

Reply:
Yorumlarınız için teşekkürler.